Guardfile

```diff
@@ -0,0 +1,70 @@
+# A sample Guardfile
```

Comment:
Eric, it's a gem that'll automatically watch and run specs against any file you've made changes to.